content/sensu-core/2.0/overview/silencing.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Intro to Silencing"
+date: 2018-02-22T10:49:25-08:00
+description: 
```

Comment:
I am curious why we have a date and no description though.

Reply:
Oh, probably because I generated it and forgot to edit :)